scripts/pyro_camera_server.py

```diff
@@ -22,4 +22,5 @@
     args = parser.parse_args()
 
     # Run the camera server
-    run_camera_server(args.ignore_local, key=args.key)
+    run_camera_server(ignore_local=args.ignore_local,
+                      key=args.key)
```

Comment:
Reverted, and more, because I've taken the key parameter out.

I do have an issue with my linter plugin though. Whatever I do I can't get it to accept a custom line length, which means it flags issues all over POCS and huntsman-pocs code, making it hard to spot real issues.